### ISSUE TITLE
Treat missing data optioms

### ISSUE THREAD

**Issue by hyei** [lorenzoaiello/terraform-aws-rds-alarms#11]
I'd like to be able to set the treat_missing_data  option for the alarms while using this module. For example, I'd like to set it to notBreaching, to avoid instances where no data is captured by AWS or it erroneously marks the the alarm as insufficient data and then reports when it transitions back to OK.

**Comment by lorenzoaiello** [comment]
Hey @hyei ,

I created #26 to address this. Can you take a look and let me know if it addresses your ask?

**Comment by hyei** [comment]
Yes, it looks like it will do what I need it to, which is to prevent the alarms created in the module to all complain if I temporarily pause/shutdown the rds instance with the alarms in place. We subscribe to rds events to be notified when the DB/RDS is down and none of the alarms supported in the module replace that, so individual settings for each alarm is not needed.

**Comment by blunckr-aj** [comment]
I'm running into this with both datapoints_to_alarm and treat_missing_data.
